src/test/java/uk/gov/pay/adminusers/resources/InviteResourceCompleteTest.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.pay.adminusers.resources;
+
+import org.junit.Test;
+
+import static javax.ws.rs.core.Response.Status.CONFLICT;
+import static javax.ws.rs.core.Response.Status.GONE;
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.text.MatchesPattern.matchesPattern;
+import static uk.gov.pay.adminusers.fixtures.InviteDbFixture.inviteDbFixture;
+import static uk.gov.pay.adminusers.fixtures.UserDbFixture.userDbFixture;
+
+public class InviteResourceCompleteTest extends IntegrationTest {
+    public static final String INVITES_RESOURCE_URL = "/v1/api/invites";
+
+    @Test
+    public void shouldReturn200withDisabledInviteLinkingToCreatedUser_WhenPassedAValidInviteCode() {
+        String email = "example1@example.gov.uk";
+        String telephoneNumber = "088882345689";
+        String password = "valid_password";
+
+        String inviteCode = inviteDbFixture(databaseHelper)
+                .withTelephoneNumber(telephoneNumber)
+                .withEmail(email)
+                .withPassword(password)
+                .insertServiceInvite();
+
+        givenSetup()
+                .when()
+                .post(INVITES_RESOURCE_URL + "/" + inviteCode + "/complete")
+                .then()
+                .statusCode(OK.getStatusCode())
+                .body("_links", hasSize(1))
+                .body("_links[0].href", matchesPattern("^http://localhost:8080/v1/api/users/[0-9a-z]{32}$"))
+                .body("_links[0].rel", is("user"))
+                .body("disabled", is(true));
```

Comment:
how about asserting the user also (by following up the user URL) ? or at least check User  created using DatabaseTestHelper ?